### DISCUSSION TITLE
noob question about extruder

### DISCUSSION THREAD

**Discussion by fuccsticc** [Jyers/Marlin#915] [Q&A] [answered]
after updating to latest preconfigured 4.2 ubl 10x10 my extruder is running backwards... do i have to manually compile to change that or is there a different option? i appreciate the help

**Comment by gothcha** [comment, marked as answer]
Do you have the 4.2.2 or 4.2.7 board? Make sure you use the appropriate firmware version. The steppers are wired in opposite polarities on the two boards.

**Reply by Jyers** [reply]
you can also simply switch the pins on the cable if you don't want to compile your own.
